WebRoot/js/zimbraMail/share/view/htmlEditor/ZmHtmlEditor.js

```diff
@@ -838,6 +838,22 @@ function(id, autoFocus) {
 		menubar: false,
 		ie7_compat: false,
 		object_resizing : true,
+		pramukhime_options: {
```

Comment:
added this for the signature tinymce (because there are two tinymce init objects)